std_msgs/Header header

int32 grid_size
int8[] grid

int32 path_size 
int32 path_start
Point[] path